wallet/mixing.go

```diff
@@ -317,3 +321,104 @@ func randomInputSource(source txauthor.InputSource) txauthor.InputSource {
 		return selected, nil
 	}
 }
+
+// IsMixTx tests if a transaction is a CSPP-mixed transaction, which must have 3
+// or more outputs of the same amount, which is one of the pre-defined mix
+// denominations. mixDenom is the largest of such denominations. mixCount is the
+// number of outputs of this denomination.
+func IsMixTx(tx *wire.MsgTx) (isMix bool, mixDenom int64, mixCount uint32) {
```

Comment:
perhaps name this PossibleCoinJoin, since it's a guess. We don't know for certain if it is a mixed transaction or not with this heuristic.